apps/api-extractor/src/api/ExtractorConfig.ts

```diff
@@ -158,8 +182,9 @@ interface IExtractorConfigParameters {
   overrideTsconfig: {} | undefined;
   skipLibCheck: boolean;
   apiReportEnabled: boolean;
-  reportFilePath: string;
-  reportTempFilePath: string;
+  reportConfigs: readonly IApiReportConfig[];
+  reportDirectoryPath: string;
+  reportTempDirectoryPath: string;
```

Comment:
Rush Stack typically uses the word `folder` instead of `directoryPath` because it is much more concise.

Reply:
(And `filePath` is normally a filename that may include a relative or absolute path or not. And `fileName` is generally the name+extension part of the path.)